career-model/README.md

```diff
@@ -1,3 +1,34 @@
+# Steven's instructions 
+
+## Setup python virtual environment 
+
+1. I am going to assume that you have Python3 installed
+
+2. Execute the following in terminal: 
+
+```{bash}
+python3.10 -m venv career-model/venv                # make python virtual environment
+source career-model/venv/bin/activate               # activate virtual environment 
+pip install -r career-model/requirements.txt        # install python dependencies
+```
+
+3. Now, you can start the service using the following: 
+
+    - if you are testing, then use `% python career-model/prediction_service.py`.
+    - if you want to use in production `% python career-model/prediction_service.py --production` or `% python career-model/prediction_service.py -p` for short. 
+
+4. The following endpoints are available: 
+
+    - `127.0.0.1:4000`: a dummy "hello world!" endpoint 
+    - `127.0.0.1:4000/prediction`: endpoint handling POST request for making predictions; student info will be contained in request body
+    - `127.0.0.1:4000/docs`: UI for API documentation; you can also find schema for POST endpoint request body here 
```

Comment:
Age is bounded [18, 25] in the predict model, but for the UI, users can input negative ages without receiving a bad request error. Otherwise, looks good.

Reply:
resolved by [fe52e01](https://github.com/CMU-313/spring23-nodebb-team-c2/pull/54/commits/fe52e01f093593aa8d0176ddebaae3cfa0896d4d)